integration/affixes/google/protobuf/timestamp.ts

```diff
@@ -0,0 +1,226 @@
+// Code generated by protoc-gen-ts_proto. DO NOT EDIT.
+// source: google/protobuf/timestamp.proto
+
+/* eslint-disable */
+import { BinaryReader, BinaryWriter } from "@bufbuild/protobuf/wire";
```

Comment:
Ah interesting; @ChapterSevenSeeds do you know why these two files were generated now? Was it from your bug fix, or were they just not included on a prior PR? :thinking:

Reply:
They appeared when I followed the instructions for running the tests. I imagine your assumption that they weren't included from a prior PR is correct 😛